docs/scripts_md/tarchive_validation.md

```diff
@@ -0,0 +1,74 @@
+# NAME
+
+tarchive\_validation.pl -- Validates the tarchive against the one inserted in
+the LORIS database.
+
+# SYNOPSIS
+
+perl tarchive\_validation.pl `[options]`
+
+Available options are:
+
+\-profile     : name of the config file in ../dicom-archive/.loris-mri
+
+\-reckless    : upload data to the database even if the study protocol
+                is not defined or if it is violated
+
+\-globLocation: loosen the validity check of the tarchive allowing for
+                the possibility that the tarchive was moved to a
+                different directory
+
+\-newScanner  : boolean, if set, register new scanners into the database
+
+\-verbose     : boolean, if set, run the script in verbose mode
+
+# DESCRIPTION
+
+The program does the following validations:
+
+\- Verification of the DICOM study archive given as an argument to the script
+using the checksum from the one inserted in the database
+
+\- Verification of the PSC information using whatever field containing the site
+string (typically, the patient name or patient ID)
+
+\- Verification/determination of the ScannerID of the DICOM study archive
```

Comment:
@driusan 
do we say "determination" for the act of determining something?
to me I use determination in the context that I am determined to do something :)